ui/app/helpers/all-features.js

```diff
@@ -1,15 +1,18 @@
 import { helper as buildHelper } from '@ember/component/helper';
 
 const ALL_FEATURES = [
-  'HSM',
```

Comment:
Will this affect the order they're displayed in in the UI? If so, will the change be jarring or strange?

Reply:
It will, but it's just a list of items.  I thought it made it easier to navigate.  Let me ping @joshuaogle though to make sure this change is OK.  Joshua, is making this list of license features in alphabetical order OK, or is there a specific list order I should follow?

![image](https://user-images.githubusercontent.com/6618863/77553210-212ec480-6e7a-11ea-8fb8-ed2228efa8d3.png)